### PR TITLE
only pivot

### PR DESCRIPTION
**Pull request by greenblitz4590**
dont merge until all elbow , pivot, command aprovved, wrist

### CODE REVIEW

**Review by YoniKiriaty — APPROVED** [review]
I really dont like you deleting everything but ok

File: src/main/java/edu/greenblitz/robotName/subsystems/shooter/Pivot/PivotConstants.java [hunk 1]
```diff
@@ -28,7 +41,9 @@ public class PivotConstants {
 
     public static final double kA = 1;
 
-    public static final double TOLERANCE = Units.degreesToRadians(2);
+    public static final double TOLERANCE = Units.degreesToRadians(0.5);
+
+    public static final double NO_COLLISION_DELAY_SECONDS = 0.7;
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/shooter/Pivot/PivotConstants.java:46]:
```suggestion
    public static final double DELAY_FOR_NO_COLLISION_SECONDS = 0.7;
```
or smth

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

**Review by bossyshrimp68 — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/subsystems/shooter/Pivot/Pivot.java [hunk 2]
```diff
@@ -60,25 +62,30 @@ public void setIdleMode(NeutralModeValue idleMode) {
...
 
-    public void moveToAngle(double goalAngle) {
-        pivot.moveToAngle(goalAngle);
+    public void moveToAngle(Rotation2d targetAngle) {
+        pivot.moveToAngle(targetAngle);
     }
 
     public void standInPlace() {
-        pivot.setPower(getStaticFF());
+        if (RobotConstants.ROBOT_TYPE == Robot.RobotType.SIMULATION) {
```
Comment (itamaroryan) [inline src/main/java/edu/greenblitz/robotName/subsystems/shooter/Pivot/Pivot.java:74]: Why not use `isSimulation()`

**Review by itamaroryan — CHANGES_REQUESTED** [review]
The CR grindset never stops

**Review by bossyshrimp68 — APPROVED** [review]

**Review by itamaroryan — APPROVED** [review]
Massive improvement, very cool!

**Pull request merged**